src/controllers/index.ts

```diff
@@ -53,21 +53,22 @@ export * as updateConfirmation from "./update/confirmation.controller";
 export * as overseasEntityReview from "./update/overseas.entity.review.controller";
 export * as updateBeneficialOwnerBoMoReview from "./update/update.beneficial.owner.bo.mo.review.controller";
 export * as overseasEntityUpdateDetails from "./update/overseas.entity.update.details.controller";
+export * as updateBeneficialOwnerStatements from "./update/update.beneficial.owner.statements.controller";
 export * as updateBeneficialOwnerType from "./update/beneficial.owner.type.controller";
+export * as updateBeneficialOwnerIndividual from "./update/update.beneficial.owner.individual.controller";
+export * as updateReviewBeneficialOwnerIndividual from "./update/update.review.beneficial.owner.individual";
+export * as updateBeneficialOwnerOther from "./update/update.beneficial.owner.other.controller";
 export * as updateReviewBeneficialOwnerOther from "./update/update.review.beneficial.owner.other";
 export * as updateBeneficialOwnerGov from "./update/update.beneficial.owner.gov.controller";
-export * as updateBeneficialOwnerStatements from "./update/update.beneficial.owner.statements.controller";
+export * as updateReviewBeneficialOwnerGov from "./update/update.review.beneficial.owner.gov";
 export * as updateManagingOfficerIndividual from "./update/update.managing.officer.controller";
-export * as updateBeneficialOwnerIndividual from "./update/update.beneficial.owner.individual.controller";
+export * as updateReviewIndividualManagingOfficer from "./update/update.review.individual.managing.officer";
+export * as updateManagingOfficerCorporate from "./update/update.corporate.managing.officer.controller";
+export * as updateReviewManagingOfficerCorporate from "./update/update.review.managing.officer.corporate.controller";
 export * as confirmToRemove from "./update/confirm.to.remove.controller";
 export * as updateSignOut from "./update/update.sign.out.controller";
-export * as updateManagingOfficerCorporate from "./update/update.corporate.managing.officer.controller";
-export * as updateBeneficialOwnerOther from "./update/update.beneficial.owner.other.controller";
 export * as updateFilingDate from "./update/update.filing.date.controller";
 export * as updateRegistrableBeneficialOwner from "./update/update.registrable.beneficial.owner.controller";
 export * as resumeUpdateSubmission from "./update/update.resume.submission.controller";
-export * as updateReviewBeneficialOwnerIndividual from "./update/update.review.beneficial.owner.individual";
-export * as updateReviewBeneficialOwnerGov from "./update/update.review.beneficial.owner.gov";
-export * as updateReviewIndividualManagingOfficer from "./update/update.review.individual.managing.officer";
 export * as updateContinueSavedFiling from "./update/update.continue.saved.filing.controller";
 export * as updateReviewOverseasEntityInformation from "./update/update.review.overseas.entity.information.controller";
```

Comment:
just re-arranged the ordering of things in here to more align with the order the pages appear.